MoltenVK/MoltenVK/Commands/MVKMTLBufferAllocation.h

```diff
@@ -100,6 +102,7 @@ class MVKMTLBufferAllocationPool : public MVKObjectPool<MVKMTLBufferAllocation>
     MTLStorageMode _mtlStorageMode;
 	MVKSmallVector<id<MTLBuffer>, 64> _mtlBuffers;
     MVKDevice* _device;
+	bool _isThreadSafe;
```

Comment:
This isn't used. I don't see why we need this.

Reply:
It's used a few lines above there in `returnAllocation()`. It handles the whether to return/return-safely, and is the compliment to the same flag that handles acquire/aquire-safely in `MVKMTLBufferAllocator`

Reply:
Oh, sorry. How did I miss that? Normally, I'm more observant than that...